joomla-projects/j4adminui
# Added missing translations "Clear Expired Cache" into mod_menu langua…

Pull Request for Issue #105 .

### Summary of Changes
Added missing translations into language file


### Testing Instructions
Click on System > Maintenance on the sidebar and notice that it should not contain the constant `MOD_MENU_PURGE_EXPIRED_CACHE`
